src/iexamine.cpp

```diff
@@ -4052,23 +4077,24 @@ void iexamine::pay_gas( player &p, const tripoint &examp )
         popup( _( "You're illiterate, and can't read the screen." ) );
     }
 
-    int pumpCount = getNearPumpCount( examp );
+    std::string fuelType;
+    int pumpCount = getNearPumpCount( examp, fuelType );
     if( pumpCount == 0 ) {
-        popup( str_to_illiterate_str( _( "Failure!  No gas pumps found!" ) ) );
+        popup( str_to_illiterate_str( _( "Failure!  No " + fuelType + " pumps found!" ) ) );
```

Comment:
That probably won't be translated. You probably need to use something like this:

```suggestion
        popup( str_to_illiterate_str( string_format( _( "Failure!  No %s pumps found!"), fuelType ) ) );
```

Reply:
Thank you, will fix this now.